public/Copy-DbaDatabase.ps1

```diff
@@ -298,6 +301,12 @@ function Copy-DbaDatabase {
             $ConfirmPreference = 'none'
         }
 
+        if (-not $InputObject -and -not $Source) {
```

Comment:
Can you please move that up and make it the first test? We try to test main inputs first and then additional parameters.

Reply:
I changed that.